app/controllers/state_file/questions/ny_w2_controller.rb

```diff
@@ -55,6 +55,11 @@ def load_w2
         @w2 = @w2s.detect { |w2| w2.w2_index == w2_index }
       end
 
+      def prev_path
+        return path_for_step(self.class) if action_name == "edit"
```

Comment:
exactly the code i had written lol. can we add this to UnemploymentController as well?